src/utils.js

```diff
@@ -8,6 +8,20 @@ import Image from 'components/Image';
 import Anchor from 'components/Anchor';
 
 
+// https://formatjs.io/docs/react-intl/api/#createintl
+const intl = createIntl({locale: 'nl-NL',}, createIntlCache());
```

Comment:
Is it okay to hardcode the locale like this?  If no locale is specified an error is logged and says that it'll fallback to english.  I'm not sure if we want to make the locale configurable at this point or if we see it as outside the scope of the MVP.

Reply:
as discussed on Slack - the `useIntl()` hook call should be done in the component and passed as argument to the utility functions.